presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterFactory.java

```diff
@@ -246,7 +246,7 @@ else if (com.facebook.hive.orc.OrcOutputFormat.class.getName().equals(storageFor
                     dwrfWriterEncryption));
         }
         catch (IOException e) {
-            throw new PrestoException(HIVE_WRITER_OPEN_ERROR, "Error creating " + orcEncoding + " file", e);
+            throw new PrestoException(HIVE_WRITER_OPEN_ERROR, "Error creating " + orcEncoding + " file. " + e.getMessage(), e);
```

Comment:
`e` is already an throwable that will have its stack trace appended after `PrestoException`. Do we still need to add `e.getMessage()` here as it won't provide more info?

Reply:
Is it because the stack trace could be too long that the `e` part will be truncated?

Reply:
Only error message is logged in scuba. Also the first line of failed chronos job shows just the error message. So having the storage error message in the final error message is helpful for debugging as "Error creating DWRF file" as error message is cryptic and not clear.  Ex: 20211113_211617_70505_db3n4